ethcore/src/executive.rs

```diff
@@ -346,6 +346,10 @@ impl<'a> CallCreateExecutive<'a> {
 				| Err(vm::Error::OutOfBounds)
 				| Err(vm::Error::Reverted)
 				| Ok(FinalizationResult { apply_state: false, .. }) => {
+					let ripemd = Address::from_low_u64_be(3);
```

Comment:
If we agree that this is a right place for this 'fix', I'll add comments and move this value to the constant